pkg/services/authz/zanzana/common/tuple.go

```diff
@@ -245,3 +249,21 @@ func ToOpenFGATuples(tuples []*authzextv1.Tuple) []*openfgav1.Tuple {
 	}
 	return result
 }
+
+func AddRenderContext(req *openfgav1.CheckRequest) {
+	if req.ContextualTuples == nil {
+		req.ContextualTuples = &openfgav1.ContextualTupleKeys{}
+	}
+	if req.ContextualTuples.TupleKeys == nil {
+		req.ContextualTuples.TupleKeys = make([]*openfgav1.TupleKey, 0)
+	}
+
+	req.ContextualTuples.TupleKeys = append(req.ContextualTuples.TupleKeys, &openfgav1.TupleKey{
+		User:     req.TupleKey.User,
```

Comment:
@kalleep if we check prefix, then we have to build user dynamically. Do you think it's safe to have it here, or should we force it using `render` and extracted user UID instead?

Reply:
It should be fine line this.